index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Best Gaming Event 2023</title>
+    <link rel="stylesheet" href="./static/base_style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <header>
+        <div id="modal__container" class="modal__container">
+            <div class="modal__header">
+              <div></div>
+              <i id="btn__close" class="fa fa-times modal__icon" aria-hidden="true"></i>
+            </div>
+            <div class="modal__content">
+              <a href="About.html"  id="modal__titles1"> About</a>
+              <a href="about.html"  id="modal__titles2"> Program</a>
+              <a href="#contact" id="modal__titles3"> Join</a>
+              <a href="#contact" id="modal__titles3"> Sponsor</a>
+              <a href="#contact" id="modal__titles3"> News</a>
+            </div>
+        </div>
+        <i id="btn__open" class="fa fa-bars" aria-hidden="true"></i>
+      <nav class="socail_nav hide">
+        <div class="socail_nav_div">
+          <a href="#"
+            ><img src="./images/facebook.png" alt="facebook_icon"
+          /></a>
+          <a href="#"><img src="./images/twitter.png" alt="twitter_icon" /></a>
```

Comment:
- [ ] Great design you got there. Please, as mentioned clearly in your project guidelines, this is someone's design(Cindy shin) you are suppose to personalize the content but not change the design, there fore Colors.
Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
,Layout: composition and space between elements should be followed. 
- please use a background almost similar to the gray color,
- your titles should follow the design color and weight, and they should be well positioned as well, as in the original design. Consider this as a means of proving to a client that you can replicate what he or she asks of you. 
- kindly implement this for mobile/desktop and the two pages. 

yours             |  original
:-------------------------:|:-------------------------:
![Screenshot (1001)](https://user-images.githubusercontent.com/65298048/234512487-c26413a8-60a3-479b-8c49-833dd200b70c.png)  |  ![Screenshot (1002)](https://user-images.githubusercontent.com/65298048/234512711-62ebd86a-3e82-4a32-9a3f-b4e7f0c9e3cc.png)